src/engine/sys/sdl_glimp.cpp

```diff
@@ -575,8 +572,29 @@ static rserr_t GLimp_SetMode( int mode, bool fullscreen, bool noborder )
 		logger.Warn( "GLEW version < 2.0.0 doesn't support GL core profiles" );
 	}
 
+	int colorBits = std::max( 0, r_colorbits->integer );
+	// int alphaBits = std::max( 0, r_alphabits->integer );
+	int depthBits = std::max( 0, r_depthbits->integer );
+	int stencilBits = std::max( 0, r_stencilbits->integer );
+	int latched_samples = std::max( 0, r_ext_multisample->integer );
+
+	if ( colorBits == 0 )
+	{
+		colorBits = 24;
+	}
+
+	if ( colorBits > 24 )
+	{
+		logger.Warn( "Color bitness %d set by r_colorbits can't be greater to 24, fallback to 24", colorBits );
+		colorBits = 24;
+	}
+
+	int samples;
+
 	do
```

Comment:
Better review the whole PR as a single commit because this is already removed, and GitHub already mark this comment as outdated without me pushing anything.

Reply:
If you don't want some commits to be reviewed, then squash those I guess?

Reply:
The problem is that it's like the rewrite steps of an equation in order to reduce it to solve it. Some steps are done with the only purpose to move things around in a way the behavior must be the same (that's why some unused stuff may still be there) but makes easier to do the actual stuff after that. The big problem in squashing those steps is that it makes impossible to bisect the various steps in case a mistake was introduced at some point… The bisect would just identify the squashed commit, making hard to identify the cause. Exactly like reducing an equation after having wrongly assumed something is equal, and not having reported that assumption was made…

Reply:
There is many PR were I would expect separate steps to be reviewed as single changes, but for this special PR I wrote in purpose:

> The various commits reflect the iterative work I did in rewriting that code. So there is a rewrite… that is entirely rewritten again after that. I'll probably squash in the end, but keeping those commit separated can help to debug some things (like to figure out at which point of my effort I may have introduced an issue if I did). So, better review the changes as if it was a single commit.